### PR TITLE
Prepare release 1.0.0

### PR DESCRIPTION
**Pull request by awjh-ibm**
- Sets version number as 1.0.0
- Updates read me to make clear of fabric 2.0 support
- Updates azure to run build on tagged

### CODE REVIEW

File: .azure-pipelines/azure-pipelines.yml [hunk 1]
```diff
@@ -7,6 +7,9 @@ trigger:
   branches:
     include:
       - master
+  tags:
+    include:
+      - '*'
```
Comment (heatherlp) [inline .azure-pipelines/azure-pipelines.yml]: why are you adding this now?

File: .github/RELEASE_CHECKLIST.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Release Checklist
 
 Before releasing ensure that you have:
-- Decided on a release tag. The repo uses a 3 number semantic versioning system and therefore the tag should be of the form v.X.X.X e.g. v2.0.0. This value will be known in the rest of this document as `<RELEASE_TAG>`.
+- Decided on a release tag. The repo uses a 3 number semantic versioning system and therefore the tag should be of the form v.X.X.X e.g. v1.0.0. This value will be known in the rest of this document as `<RELEASE_TAG>`.
```
Comment (heatherlp) [inline .github/RELEASE_CHECKLIST.md]: Can you change this sentence to say "decide" at the start, please

File: .azure-pipelines/azure-pipelines.yml (hunk 1 above)
Comment (awjh-ibm) [reply .azure-pipelines/azure-pipelines.yml]: I relealised we were missing it when it didn't run on the previous tagged build

File: .github/RELEASE_CHECKLIST.md (hunk 2 above)
Comment (awjh-ibm) [reply .github/RELEASE_CHECKLIST.md]: But it start with you have....

**Pull request merged**